aci_interface_policies.tf

```diff
@@ -434,3 +434,58 @@ module "aci_spine_fabric_interface_configuration" {
   shutdown     = each.value.shutdown
   role         = each.value.role
 }
+
+locals {
+  interface_state = flatten([
+    for node in local.nodes : [
+      for interface in try(node.interfaces, []) : {
+        key      = format("%s/%s/%s", node.id, try(interface.module, local.defaults.apic.interface_policies.nodes.interfaces.module), interface.port)
+        node_id  = node.id
+        module   = try(interface.module, local.defaults.apic.interface_policies.nodes.interfaces.module)
+        port     = interface.port
+        shutdown = try(interface.shutdown, local.defaults.apic.interface_policies.nodes.interfaces.shutdown)
+      }
+    ]
+    if(try(local.apic.auto_generate_switch_pod_profiles, local.defaults.apic.auto_generate_switch_pod_profiles) || try(local.apic.auto_generate_access_leaf_switch_interface_profiles, local.defaults.apic.auto_generate_access_leaf_switch_interface_profiles) || try(local.apic.auto_generate_access_spine_switch_interface_profiles, local.defaults.apic.auto_generate_access_spine_switch_interface_profiles)) && (length(var.managed_interface_policies_nodes) == 0 || contains(var.managed_interface_policies_nodes, node.id)) && try(local.apic.new_interface_configuration, local.defaults.apic.new_interface_configuration) == false
```

Comment:
You can remove the `(length(var.managed_interface_policies_nodes) == 0 || contains(var.managed_interface_policies_nodes, node.id))` code since that's already tested in the creation of the `local.nodes` variable in `main.tf`.

Reply:
Fixed this.